### ISSUE TITLE
Bad browser performance on large text with highlighted features

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#95]
Scrolling in the notebook is noticeably laggy when the text with highlighted features is in sight. Maybe it will help if we merge spans with the same weight (at least in case of word analyzers).

**Comment by kmike** [comment]
Yeah, I also saw performance issues, epecially in IPython notebook. Profile showed that browser was recalculating element heights or something like that. I wonder if that's specific to IPython.

**Comment by lopuhin** [comment]
mostly fixed in #115
